src/Microsoft.Identity.Client/Cache/TokenCacheJsonSerializer.cs

```diff
@@ -19,35 +41,40 @@ public TokenCacheJsonSerializer(ITokenCacheAccessor accessor)
 
         public byte[] Serialize()
         {
-            var cache = new CacheRoot();
-            foreach (var t in _accessor.GetAllAccessTokens())
+            var cache = new CacheSerializationContract();
+            foreach (var token in _accessor.GetAllAccessTokens())
             {
-                cache.AccessTokens[t.GetKey().ToString()] = t;
+                cache.AccessTokens[token.GetKey()
+                                    .ToString()] = token;
             }
 
-            foreach (var t in _accessor.GetAllRefreshTokens())
+            foreach (var token in _accessor.GetAllRefreshTokens())
             {
-                cache.RefreshTokens[t.GetKey().ToString()] = t;
+                cache.RefreshTokens[token.GetKey()
+                                     .ToString()] = token;
             }
 
-            foreach (var t in _accessor.GetAllIdTokens())
+            foreach (var token in _accessor.GetAllIdTokens())
             {
-                cache.IdTokens[t.GetKey().ToString()] = t;
+                cache.IdTokens[token.GetKey()
+                                .ToString()] = token;
             }
 
-            foreach (var t in _accessor.GetAllAccounts())
+            foreach (var acct in _accessor.GetAllAccounts())
```

Comment:
>acct [](start = 25, length = 4)

accounts? does this need to be abbreviated?

Reply:
#resolved